apps/MiUnlockTool/install

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+DIRECTORY="$(dirname "$(dirname "$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )")")"
+
+function error {
+echo -e "\\e[91m$1\\e[39m"
+exit 1
+}
```

Comment:
@Botspot correct me if I'm wrong, but I think the `DIRECTORY` variable and `error` function don't have to be declared anymore.